blocks/editable/style.scss

```diff
@@ -56,6 +56,19 @@
 	}
 }
 
+.editor-visual-editor__block:not( .is-selected ) .blocks-editable.drop-cap-true {
```

Comment:
contentEditable cannot handle styling `:first-letter`, also when editing it is nicer to be able to see the whole line.

Agreed on moving to styles. Having some conversations on how to go about front-end styles in general.